server/db/models/index.js

```diff
@@ -1,17 +1,20 @@
 const Conversation = require("./conversation");
 const User = require("./user");
 const Message = require("./message");
+const Participant = require("./participant");
 
 // associations
 
-User.hasMany(Conversation);
-Conversation.belongsTo(User, { as: "user1" });
-Conversation.belongsTo(User, { as: "user2" });
-Message.belongsTo(Conversation);
+Conversation.belongsToMany(User, { through: Participant });
+Conversation.belongsToMany(User, {through: Participant, as: 'filterUser'});
 Conversation.hasMany(Message);
+Conversation.hasMany(Participant);
```

Comment:
```suggestion
Conversation.belongsToMany(User, {through: Participant});
Users.belongsToMany(Conversation, {through: Participant});
Conversation.hasMany(Message);
```

Reply:
Oooh. You're so very correct sir. Very elegant.